src/sentry/conf/server.py

```diff
@@ -920,7 +920,7 @@ def create_partitioned_queues(name):
     "organizations:filters-and-sampling": False,
     # Enable Dynamic Sampling errors in the org settings
     "organizations:filters-and-sampling-error-rules": False,
-    # Allow organizations to configure built-in symbol sources.
+    # Allow organizations to configure external symbol sources.
```

Comment:
This is wrong, this feature flag controls built-in sources. The other feature flag, `organizations:custom-symbol-sources` controls external symbol sources, see its comment.

Reply:
but according to the code, we do not display custom Symbol Sources and Built-in Sources without this flag

Reply:
we talked offline and agreed that "Allow organizations to configure all symbol sources." would be a good description